pkg/webservice/dao/partition_info.go

```diff
@@ -35,18 +35,10 @@ type PartitionInfo struct {
 
 type PartitionCapacity struct {
 	Capacity     string `json:"capacity"`
-	UsedCapacity string `json:"usedCapacity"`
+	UsedCapacity string `json:"usedcapacity"`
```

Comment:
Renaming as before because this change should not break the existing use cases. Once new API has been adopted completely, we can do the naming with similar convention.